docs/how-to/hip_runtime_api/memory_management/device_memory.rst

```diff
@@ -0,0 +1,52 @@
+.. meta::
+  :description: This chapter describes the device memory of the HIP ecosystem
+                ROCm software.
+  :keywords: AMD, ROCm, HIP, device memory
+
+.. _device_memory:
+
+*******************************************************************************
+Device memory
+*******************************************************************************
+
+Device memory exists on the device, e.g. on GPUs in the video random access
+memory (VRAM), and is accessible by the kernels operating on the device. Recent
+architectures use graphics double data rate (GDDR) synchronous dynamic
+random-access memory (SDRAM) such as GDDR6, or high-bandwidth memory (HBM) such
+as HBM2e. Device memory can be allocated as global memory, constant, texture or
+surface memory.
+
+Global memory
+================================================================================
+
+Read-write storage visible to all threads on a given device. There are
+specialized versions of global memory with different usage semantics which are
+typically backed by the same hardware, but can use different caching paths.
+
+Constant memory
+================================================================================
+
+Read-only storage visible to all threads on a given device. It is a limited
+segment backed by device memory with queryable size. It needs to be set by the
+host before kernel execution. Constant memory provides the best performance
+benefit when all threads within a warp access the same address.
+
+Texture memory
+================================================================================
+
+Read-only storage visible to all threads on a given device and accessible
+through additional APIs. Its origins come from graphics APIs, and provides
+performance benefits when accessing memory in a pattern where the
+addresses are close to each other in a 2D representation of the memory. 
+
+The texture management module of HIP runtime API contains the functions of
+texture memory.
```

Comment:
Add a link to the reference. Also specify reference in the sentence, as now there are two "HIP runtime API"s in the toc
```suggestion
The texture management module of the HIP runtime API reference contains the functions of
texture memory.
```

Reply:
Fixed.